src/AspNetCoreModuleV2/CommonLib/FileOutputManager.cpp

```diff
@@ -49,6 +49,10 @@ FileOutputManager::Start()
     STRU struPath;
     FILETIME processCreationTime;
     FILETIME dummyFileTime;
+    
+    // To make Console.* functions work, allocate a console
+    // in the current process.
+    AllocConsole();
```

Comment:
Check return value?

Reply:
I'll check that it returns an error where there is already a console present.

Reply:
Also, I'm thinking of conditionally calling FreeConsole if AllocConsole succeeded.